src/Drawing.js

```diff
@@ -29,28 +30,22 @@ class Drawing {
         this.headers = {};
         this.tables = {};
         this.blocks = {};
-        this.handleCount = 0;
-
-        this.ltypeTableHandle = this._generateHandle();
-        this.layerTableHandle = this._generateHandle();
-        this.blockRecordTableHandle = this._generateHandle();
 
         this.dictionary = new Dictionary();
-        this._assignHandle(this.dictionary);
 
         this.setUnits("Unitless");
 
-        for (const lineType of Drawing.LINE_TYPES) {
```

Comment:
What is reason for this? How is "forEach" better than "for" loop?

Reply:
Yeah just syntax sugar